rfc/5/versions/1/index.md

```diff
@@ -0,0 +1,977 @@
+- Additional transformations for single images MUST be stored under a field `coordinateTransformations`
+  in the multiscales dictionaries.
+  This `coordinateTransformations` field MUST contain a list of valid [transformations](#transformation-types).
+- Transformations between two or more images MUST be stored in the attributes of a parent zarr group.
+  For transformations that store data or parameters in a zarr array,
+  those zarr arrays SHOULD be stored in a zarr group `coordinateTransformations`.
+
+Implementations SHOULD prefer to store transformations as a sequence of less expressive transformations
+(i.e., sequence[translation, rotation] instead of affine transformation with translation/rotation) component. 
+
+<pre>
+store.zarr                      # Root folder of the zarr store
+│
+├── zarr.json                   # coordinate transformations describing the relationship between two image coordinate systems
+│                               # are stored in the attributes of their parent group.
+│                               # transformations between 'volume' and 'crop' coordinate systems are stored here.
+│
+├── coordinateTransformations   # transformations that use array storage go in a "coordinateTransformations" zarr group.
+│   └── displacements           # for example, a zarr array containing a displacement field
+│       └── zarr.json
+│
+├── volume
+│   ├── zarr.json              # group level attributes (multiscales)
+│   └── 0                       # a group containing the 0th scale
+│       └── image               # a zarr array
+│           └── zarr.json      # physical coordinate system and transformations here
+└── crop
+    ├── zarr.json              # group level attributes (multiscales)
+    └── 0                       # a group containing the 0th scale
+        └── image               # a zarr array
+            └── zarr.json      # physical coordinate system and transformations here
+</pre>
+
+````{admonition} Example
+Two instruments simultaneously image the same sample from two different angles,
+and the 3D data from both instruments are calibrated to "micrometer" units.
+Two samples are collected ("sampleA" and "sampleB").
+An analysis of sample A requires measurements from both instruments' images at certain points in space.
+Suppose a region of interest (ROI) is determined from the image obtained from instrument 2,
+but quantification from that region is needed for instrument 1.
+Since measurements were collected at different angles,
+a measurement by instrument 1 at the point with coordinates (x,y,z)
+may not correspond to the measurement at the same point in instrument 2
+(i.e., it may not be the same physical location in the sample).
+To analyze both images together, they must be in the same coordinate system.
+
+The set of coordinate transformations encodes relationships between coordinate systems,
+specifically, how to convert points and images to different coordinate systems.
```

Comment:
```suggestion
specifically, how to convert points from one coordinate system to another.
```
transforms do not direclty transform images; they transform points, which are then used to transform images (usually by interpolation on to the transformed points).